### ISSUE TITLE
Question about the pretrained checkpoint

### ISSUE THREAD

**Issue by pkqbajng** [pmj110119/RenderOcc#36]
Hi, thanks for your excellent job. However, I found that you adopted a pretrained checkpoint on the objection detection trained in supervised manner? Is it reasonable for a self-supervised task?

**Comment by pmj110119** [comment]
Thanks for attention. Since our baseline([BEVStereo-occ](https://github.com/HuangJunJie2017/BEVDet/blob/dev3.0/configs/bevdet_occ/bevdet-occ-r50-4d-stereo-24e.py)) on nuScenes utilized pre-trained detection weight, RenderOcc adopts the same setup for fair comparison. You may try training from scratch, which could lead to a slight decrease in performance after training for more epochs. You can refer to our experiments as shown in Table 3 of [arxiv](https://arxiv.org/abs/2306.09117) on nuScenes and Table 2 of [arxiv](https://arxiv.org/pdf/2309.09502.pdf)  on SemanticKitti, which didn't employ pre-trained weights for 3D detection.

It's worth mentioning that RenderOcc isn't a self-supervised setting but rather avoids the laborious creation of 3D Occupancy Labels. We still utilize easily obtainable labels such as segmentation and depth. For self-supervised occupancy methods, you can refer to recent works like OccNerf and SelfOcc.

**Comment by rmarcuzzi** [comment]
Hi!
I was also trying to reproduce the results of training only with 2d labels from scratch and achieved 19.54 mIoU. Acording to the tables you mentioned from the paper, the result on Nuscenes training from scratch is 20.2 mIoU? and in SemanticKITTI 8.24?
Thank you!

**Comment by pmj110119** [comment]
Yes, we achieved 20.2 on nuScenes and 8.24 on SemanticKITTI. 

The earlier experiment was conducted with a very early version of our model ([UniOcc](https://arxiv.org/abs/2306.09117)), and there are some untraceable parameter differences between it and the current code. You might need to adjust the learning rate and train for more epochs.
